CMakeLists.txt

```diff
@@ -487,6 +487,51 @@ if (CODECOVERAGE)
   add_link_options(--coverage)
 endif (CODECOVERAGE)
 
+# TAP test support
+option(TAP "Enable TAP test support" ON)
+
+if (TAP)
+  include(FindPerlLibs)
+  find_package(PerlLibs 5.8)
```

Comment:
You don't need to include it first, and you don't really need to know where the libraries are, only where `perl` executable is.
```suggestion
  find_package(Perl 5.8)
```

Reply:
I use PerlLibs since it sets up a number of variables that are later used. Those are not set when looking for just Perl.

Reply:
Which are those?

Reply:
https://cmake.org/cmake/help/latest/module/FindPerlLibs.html

See tsl/test/CMakeLists.txt

Reply:
Removed the need for extra paths, and thus PerlLibs check, since Nikhil confirmed it works for him without it.